test/performance/array/distCreate-large.ml-perf.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: blockDom:, blockArr:, cyclicArr:, cyclicDom:, blockCyclicDom:, blockCyclicArr:
```

Comment:
nit, but can you swap the  `cyclicArr` and `cyclicDom` order so it's consistent with the others?